doc/user-guide/Getting-started.md

```diff
@@ -1,60 +1,75 @@
 ## Installation
 
-### Getting started with MongooseIM on Mac OS X
+### Getting started with MongooseIM on Mac
 
-This short guide will show you how to compile MongooseIM from source code or install binaries from a DMG image on Mac OS X, and get users chatting right away.
+This short guide will show you how to compile MongooseIM from source code or install binaries from a DMG image on Mac, and get users chatting right away.
```

Comment:
In this short guide we will set MongooseIM up and get your users chatting right away.
You can either compile everything from the source code or install binaries from a DMG image on Mac.